static/usr/lib/core/handle-writable-paths

```diff
@@ -141,6 +141,29 @@ handle_writable_paths()
 			continue
 		fi
 	done
+
+        # now apply the defaults on top
+        handle_writable_defaults
```

Comment:
I think these two lines have inconsistent indenting (using spaces when rest uses tabs).

Reply:
thanks, fixed and pushed